src/main/resources/conf/config.yml

```diff
@@ -80,5 +80,73 @@ mbeans:
     mbeanKeys: ["type"]
     metrics:
       include:
-        - HeapMemoryUsage.committed : "Heap Memory Usage|Committed"
+        - name: "HeapMemoryUsage.committed"
+          alias: "Heap Memory Usage|Committed"
+        - name: "HeapMemoryUsage.used"
+          alias: "Heap Memory Usage|Used"
 
+  - objectName: "org.apache.activemq.metrics:type=ClientRequest,scope=Read,name=Latency"
+    mbeanKeys: ["type", "scope","name"]
+    metrics:
+      include:
+        # Normal Metric
+        - name: "Max"
+          alias: "Max"
+          aggregationType: "OBSERVATION"
+          timeRollUpType: "AVERAGE"
+          clusterRollUpType: "INDIVIDUAL"
+
+        # Composite Metric
+        - name: "HeapMemoryUsage.max"
+          alias: "Heap Memory Usage|Max"
+          multiplier: "10"
+          delta: false
+
+        # Map Metric Level 1
+        - name: "MapOfString.key1"
+          alias: "Map 1|Key 1"
+        - name: "MapOfString.key2"
+          alias: "Map 1|Key 2"
+          aggregationType: "OBSERVATION"
+          timeRollUpType: "AVERAGE"
+          clusterRollUpType: "INDIVIDUAL"
+
+        # Map Metric Level 2
+        - name: "MapOfString.map2.key2"
+          alias: "Map 1|Map 2|Key 2"
+
+        # Map Metric Level 3
+        - name: "MapOfString.map2.map3.key32"
+          alias: "Map 1|Map 2|Map 3|Key 32"
+          multiplier: "20"
+          delta: false
+          aggregationType: "OBSERVATION"
+          timeRollUpType: "AVERAGE"
+          clusterRollUpType: "INDIVIDUAL"
+          convert : {
+            "ENDANGERED" : "1",
+            "NODE-SAFE" : "2",
+            "MACHINE-SAFE" : "3"
+          }
+
+        # List Metrics
+        - name: "listOfString.metric one"
+          alias: "List Of String|metric one"
+        - name: "listOfString.metric two"
+          alias: "List Of String|metric two"
+        - name: "listOfString.metric three"
+          alias: "List Of String|metric three"
+
+
+# Note: If the current "metricCharacterReplacer" is not working on your current machine,
```

Comment:
The `metricCharacterReplacer` in the comment should be changed to `metricPathReplacements`.

Reply:
done!